PubNub/Core/PubNub+Presence.m

```diff
@@ -341,8 +342,10 @@ - (void)setConnected:(BOOL)connected
             [self heartbeatWithCompletion:block];
             [self.heartbeatManager startHeartbeatIfRequired];
         } else {
+            [self cancelSubscribeOperations];
             [self.subscriberManager unsubscribeFromChannels:presenceChannels
                                                      groups:presenceChannelGroups
+                                          informingListener:NO
```

Comment:
Why `informingListener == NO` ?

Reply:
@parfeon Could you explain a little? Would be good to get this merged at out for Atlassian.

Reply:
@jzucker2 @madisonsmith 

This actually work as I planed only for channels added by new presence API.
To report back, new API use callback blocks. There is no need to report to state listeners, what we unsubscribed from channel, which wasn't part of subscription loop.
So we report back with completion block and nothing fired to event listeners.